src/merkle_tree.cairo

```diff
@@ -122,235 +122,6 @@ mod tests {
         assert_eq!(merkle_root(ref txids), expected_merkle_root);
     }
 
-    #[test]
```

Comment:
Why this test is removed?

Reply:
It was the same test as test4 with the same values, but directly formatted in Hash without going through the .into(), to show the difference in performance, I had asked which of the two to keep, but the 2 were left. So I deleted this one so that all the tests would have the same format.